mpas_analysis/obs/observational_datasets.xml

```diff
@@ -1479,6 +1479,70 @@
     </nameInDocs>
   </observation>
 
+  <observation>
+    <name>
+      OSNAP Temperature Salinity and Velocity Climatology
+    </name>
+    <component>
+      ocean
+    </component>
+    <description>
+<<<<<<< HEAD
+      OSNAP is designed to provide a continuous record of the full-water 
+=======
+      OSNAP was designed to provide a continuous record of the full-water 
+>>>>>>> 300a3f1b (PR review changes)
```

Comment:
This is presumably left over from a rebase?

Reply:
oops, let me check that!

Reply:
🙌